pkg/handlers/systems.go

```diff
@@ -13,30 +16,31 @@ type marshal func(interface{}) ([]byte, error)
 type SystemsListHandler struct {
 	FetchSystems fetchSystems
 	Marshal      marshal
+	Logger       *zap.Logger
```

Comment:
We can optionally define these as interfaces, but I've yet to see this pay off in past code bases. Discussion is here: https://trussworks.slack.com/archives/CLLEZMKKP/p1582139044001500